src/Components/Testimonials/TestimonialsForm.js

```diff
@@ -1,32 +1,96 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import {useEffect, useState} from 'react';
+import {Formik, Form, Field} from 'formik';
+import { ErrorsForm } from '../common/messagesForm';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import { UrlInput } from '../common/getUrlInputFile';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { validateFormTestimonials, replaceCKeditor } from '../common/validateFormTestimonials';
+import {getTestimonial,createTestimonial,modifiedTestimonial} from '../../Services/testimonialService';
+const TestimonialForm = ({testimonialId}) => {
 
-const TestimonialForm = () => {
-    const [initialValues, setInitialValues] = useState({
-       name: '',
-       description: '' 
-    });
+  const [imgState,setImgState] = useState();
+  const [dataForm, setDataForm] = useState({
+    name:'',
+    description:'',
+    image: ''
+  });
 
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'description'){
-            setInitialValues({...initialValues, description: e.target.value})
-        }
-    }
+  useEffect(() => {
+    if(testimonialId) setDataForm(getTestimonial(testimonialId));
+  }, [testimonialId]);
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        console.log(initialValues);
-    }
+  const handleChaneCkEditor = (editor, setFieldValue) => {
+    setFieldValue("description", editor.getData())
+  };
 
-    return (
-        <form className="form-container" onSubmit={handleSubmit}>
-            <input className="input-field" type="text" name="name" value={initialValues.name} onChange={handleChange} placeholder="Testimonial Title"></input>
-            <input className="input-field" type="text" name="description" value={initialValues.description} onChange={handleChange} placeholder="Testimonial description"></input>
+  const handleChangeinputFile = (e, setFieldValue) =>{ 
```

Comment:
in this case a better name maybe its 
```
const handleChangeImage = (e, setImage) =>
```
always think on whats the main goal of the function and what can be done to make clearest the code for a programmer that never read it. with current name I had to go down and read on the ui where its being used to be sure of what it do. With a name like handleChangeImage there are no dudes of that it change the image only by reading the code. That's the meaning of that the code should be enought documentation for itself.